### PR TITLE
Module Logging

### PR DESCRIPTION
**Pull request by codingwithredbull**
Added Logging to all relevant scripts, which can also be piped to a "Log" page on GUI

### CODE REVIEW

File: start.py [hunk 1]
```diff
@@ -11,14 +11,42 @@
 gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk
 from SplashScreen import SplashScreen
+import logging
 
+# create logger for entire wallet application
+logger = logging.getLogger('trtl_log')
+logger.setLevel(logging.DEBUG)
```
Comment (ftwillms) [inline start.py]: should we add the verbosity option as an argument?

**Comment by codingwithredbull** [comment]
Added additional code to show a logging panel on UI

File: start.py (hunk 1 above)
Comment (CaptainMeatloaf) [reply start.py]: I agree with this, specifying it as an argument could be useful

File: start.py [hunk 2]
```diff
@@ -11,14 +11,42 @@
...
+# create handle to log file that will hold all events
+fh = logging.FileHandler('trtl.log')
+fh.setLevel(logging.DEBUG)
 
+# --- This is the logger for CLI, we do not need it for now.
```
Comment (CaptainMeatloaf) [inline start.py]: If it's not needed, remove it and add it later when it is, leaving code commented out is bad practice unless absolutely necessary

**Pull request closed without merging**